config/localization/en/app.json

```diff
@@ -1559,6 +1559,7 @@
     "AVAILABLE_BLOCK": "Available block",
     "AVAILABLE_ON_CHAIN": "Available on {CHAIN}",
     "BALANCE_ON_CHAIN": "Balance on {CHAIN}",
+    "BANNER_CONTENT": "dYdX Chain is in Alpha - help secure the network by staking!",
```

Comment:
Do we plan on changing the `BANNER_CONTENT` text post Alpha period? Might be beneficial to rename the stringKey

Reply:
ahha i had the opposite thought that it's one less thing to change -- but u right the translation will be late to follow